File: guessingGame/guess.js
function isCorrect(guess, num){
    if (guess == num){
        console.log("congratulations")
    }
}

function isHigh(guess, num){
    if (guess > num){
        console.log("Try lower!")
    }
}

function isLow(guess, num){
    if (guess < num){
        console.log("Try Higher!")
    }
}

function guessNumber(guess){
    const num = Math.random(1, 11)
    while (true){
        if (isCorrect(guess, num)){
            break
        }
        else{
            isHigh(guess, num);
            isLow(guess, num);
        }
    }
}

File: guessingGame/guess.py
import random

num = random.choice(range(1, 11))
while True:
    guess = int(input("Guess a number between 1 and 10: "))
    if num == guess:
        print(f"congratulations! {num}")
        break
    elif guess > num:
        print("close but a bit lower")
    else:
        print("A bit higher")